### PR TITLE
feat: Add endpoint and service for user lookup

### PR DESCRIPTION
**Pull request by taustad**

### CODE REVIEW

**Review by DanielBohme — APPROVED** [review]
LGTM

File: datasheetapi/Services/Fusion/FusionPeopleService.cs [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+
+        foreach (var contextRelation in contextRelations)
+        {
+            if (contextRelation.Type == FusionContextType.OrgChart)
+            {
+                orgChartId = contextRelation.ExternalId?.ToString();
+            }
+        }
+
+        if (string.IsNullOrEmpty(orgChartId))
```
Comment (manikandan5027) [inline datasheetapi/Services/Fusion/FusionPeopleService.cs:46]: Suggestion: For better readability, its good to switch the condition. I.e, when orgChartId is present fetch it or return null.

File: datasheetapi/appsettings.json [hunk 2]
```diff
@@ -18,5 +18,11 @@
             "Microsoft.EntityFrameworkCore.Database.Command": "Warning"
         }
     },
+    "FusionPeople": {
+        "BaseUrl": "https://fusion-s-people-ci.azurewebsites.net",
```
[the remark this answers is not in the document]
Comment (manikandan5027) [reply datasheetapi/appsettings.json:22]: As discussed with Tord, This will be overwritten by azure app config, but this will be tested after merge

**Review by manikandan5027 — APPROVED** [review]
LGTM

**Pull request merged**